src/social-web/routes/feed/stage.tsx

```diff
@@ -0,0 +1,145 @@
+/**
+ * Feed Stage
+ *
+ * Main feed list view with DataViews
+ */
+
+import './style.scss';
+import { useMemo, useCallback, useState, useEffect } from '@wordpress/element';
+import { DataViews } from '@wordpress/dataviews';
+import { useView } from '@wordpress/views';
+import type { View, Field } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { Page } from '../../components/page';
+import { useFeed } from '../../hooks/use-feed';
+import {
+	titleField,
+	dateField,
+	statusField,
+	excerptField,
+	metadataField,
+	createContentField,
+} from '../../components/fields';
+import { getFeedActions } from './FeedActions';
```

Comment:
```suggestion
import { getFeedActions } from './feed-actions';
```
Among all the confusing mix of cases, filenames are in kebap-case, I think